Domain.Tests/EventSourcedRepositoryTests.cs

```diff
@@ -624,6 +627,37 @@ public void When_Refresh_is_called_on_an_aggregate_having_pending_events_it_thro
             var order = await repository.GetLatest(orderId);
 
             order.CustomerName.Should().Be("Waylon Jennings");
+        }
+
+
+        [Category("Performance")]
+        [Test, Timeout(120000)]
+        public async Task When_sourcing_an_aggregate_with_a_large_number_of_source_events_then_the_operation_completes_quickly()
+        {
+            var count = 30000;
+            var aggregateId = Guid.Parse("547E3646-DBE5-43D4-BAC9-E391336340D7");
```

Comment:
Two reasons. I didn't want to clutter up the event store on local machines since there are still some perf issues with a large event store. The second and primary reason I used it was it also served the other useful purpose of being able to comment out the db seeding part of this code after the first run, and keep re-running the test when tweaking perf; as the adding the events to the db takes ~20 seconds (compared to the ~2 seconds it takes to source the aggregate). 

There are still definitely opportunities to optimize here, as there's no reason sourcing 30k events from  the db should take so long when the in-memory sourcing is so quick (the db query is very quick - so it's not sql server's fault). So it's an optional optimization point in future (but not worth it right now - as it is comparatively a very small bottleneck in our production code.